tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/HashLeftOuterJoinExec.java

```diff
@@ -211,12 +169,16 @@ protected void loadRightToHashTable() throws IOException {
       }
 
       List<Tuple> newValue = tupleSlots.get(keyTuple);
-      if (newValue != null) {
-        newValue.add(tuple);
-      } else {
-        newValue = new ArrayList<Tuple>();
-        newValue.add(tuple);
-        tupleSlots.put(keyTuple, newValue);
+      try {
+        if (newValue != null) {
+          newValue.add(tuple.clone());
```

Comment:
I found that this cloning increases the memory space usage significantly, which causes frequent GC.
Currently, there is a problem of that the reference of the ```tuple``` value is not changed.
That is, the underlying operator returns the ```tuple``` with the same reference but the different contents.
I'll fix it.